examples/using-remark/src/pages/2016-04-15---hello-world-kitchen-sink/index.md

```diff
@@ -11,14 +11,14 @@ tags:
 ![](joshua-earle-234740.jpg)
 *Photo by [Joshua Earle](https://unsplash.com/@joshuaearle) via [Unsplash](https://unsplash.com/@joshuaearle?photo=-87JyMb9ZfU)*
 
-**_Note how all headlines below show an anchor link when you hover them?_**  
```

Comment:
Those two spaces at the end of a line actually do a <br>, so it'd be nice if you could turn off "trim end of line whitespace" – quoting from the [Markdown doc](https://daringfireball.net/projects/markdown/syntax#p): 

> When you do want to insert a <br /> break tag using Markdown, you end a line with two or more spaces, then type return.